include/poketch_data.h

```diff
@@ -0,0 +1,218 @@
+#define POKETCH_APPID_COINTOSS              14
+#define POKETCH_APPID_MOVETESTER            15
+#define POKETCH_APPID_CALENDAR              16
+#define POKETCH_APPID_DOTART                17
+#define POKETCH_APPID_ROULETTE              18
+#define POKETCH_APPID_RADARCHAINCOUNTER     19
+#define POKETCH_APPID_KITCHENTIMER          20
+#define POKETCH_APPID_COLORCHANGER          21
+#define POKETCH_APPID_MATCHUPCHECKER        22
+#define POKETCH_APPID_STOPWATCH             23
+#define POKETCH_APPID_ALARMCLOCK            24
+#define POKETCH_APPID_MAX                   25
+
+/* Max number of Map Markers. */
+#define POKETCH_MAPMARKER_COUNT             6
+
+/*
+ * Returns the size of the PoketchData object.
+ *
+ * @returns: the size of the PoketchData object.
+ */
+int Poketch_SaveSize(void);
+
+/*
+ * Initializes the given PoketchData object.
+ *
+ * @param poketchData:      The PoketchData to initialize.
+ */
+void Poketch_Init(PoketchData * poketchData);
+
+void sub_020567D0(PoketchData * poketchData);
+BOOL sub_020567E0(PoketchData * poketchData);
+
+/*
+ * Checks whether or not the given AppID is already registered in the Poketch.
+ * Valid apps are in the range 0-24.
+ *
+ * @param poketchData:      The Poketch data to check.
+ * @param appID:            The App ID to check for.
+ *
+ * @returns: TRUE if the app is registered, FALSE if not.
+ */
+BOOL PoketchData_CheckAppRegistered(PoketchData * poketchData, int appID);
+
+/*
+ * Registers the app with the given ID, making it accessible to the player.
+ *
+ * @param poketchData:      The Poketch data to check.
+ * @param appID:            The ID of the app to register (see defines above). This function asserts that the value is less than POKETCH_APPID_MAX.
+ *
+ * @returns: TRUE if the was successfully registered, FALSE if it was already registered.
+ */
+BOOL PoketchData_RegisterApp(PoketchData * poketchData, int appID);
+
+/*
+ * Gets the currently active Poketch app.
+ *
+ * @param poketchData:      The Poketch data to check.
+ *
+ * @returns: The index of the current app.
+ */
+int PoketchData_GetCurrentAppID(const PoketchData * poketchData);
+
+/*
+ * Sets the current Poketch app to the next registered app, skipping unregistered App IDs and wrapping around if necessary.
+ *
+ * @param poketchData:      The Poketch data to update.
+ *
+ * @returns: The new current app ID.
+ */
+int PoketchData_IncrementAppID(PoketchData * poketchData);
+
+/*
+ * Sets the current Poketch app to the previous registered app, skipping unregistered App IDs and wrapping around if necessary.
+ *
+ * @param poketchData:      The Poketch data to update.
+ *
+ * @returns: The new current app ID.
+ */
+int PoketchData_DecrementAppID(PoketchData * poketchData);
+
+/*
+ * Gets the current Poketch's color setting.
+ *
+ * @param poketchData:      The Poketch data to check.
+ *
+ * @returns: The Poketch's color setting.
+ */
+u32 PoketchData_GetCurrentScreenColor(const PoketchData * poketchData);
```

Comment:
We're trying to scrub instances of `Get` in function names across the repo. I'd prefer this to be `PoketchData_CurrentScreenColor`.